### DISCUSSION TITLE
可以使用xquic 作为客户端，quic-go作为服务端进行通信吗？

### DISCUSSION THREAD

**Discussion by yangqingyuan** [alibaba/xquic#131] [Q&A] [answered]
目前直接使用test_client来跟quic-go（服务端）来通讯，无法通讯，是需要做特别的改造吗？

**Comment by ruiqizhou** [comment, marked as answer]
- #58 

Retry is not yet supported.

**Reply by yangqingyuan** [reply]
那可以用其他的非xquic service端来测试吗

**Reply by Sy0307** [reply]
理论上可以，请参考 https://interop.seemann.io/

**Reply by cpp2go** [reply]
我试了，不行。能握手成功，但是收不到消息

**Reply by Sy0307** [reply]
https://interop.seemann.io/logs/2025-04-15T02:05/quic-go_xquic/http3/

您是否已经查看我分享的测试网站？

**Reply by cpp2go** [reply]
这个用的什么测试代码？

**Reply by Sy0307** [reply]
建议查看 https://interop.seemann.io/logs/2025-04-15T02:05/quic-go_xquic/http3/output.txt 里面根据不同的quic协议自己配置了docker以及命令。
同时结合 https://github.com/quic-interop/quic-interop-runner 食用。

**Reply by cpp2go** [reply]
好了，能通信了，谢谢~

**Comment by cpp2go** [comment]
现在支持了吗？

**Reply by Sy0307** [reply]
理论上可以支持的，但是可能需要按照一些参考进行配置。https://interop.seemann.io/ 供参考，里面有配置的参考。
